### ISSUE TITLE
mathjax library runs on any text between $ signs, should be $$

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#1096]
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://github.com/publiclab/plots2/labels/first-timers-only), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @jywarren to get it labelled. Or, if this is not your first time, try to solve it yourself!

****

### What happened just before the problem occurred

https://publiclab.org/notes/Willie/04-01-2014/field-testing-the-shenyei-pm-sensor#c8829

I'll add more info soon,!

### PublicLab.org username

(to help reproduce the issue)

### Browser, version, and operating system

For bug reports, fill out the above template; for feature requests, you can delete the template.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by jywarren** [comment]
Need to change default enclosure tags: http://docs.mathjax.org/en/latest/start.html#tex-and-latex-input

**Comment by jywarren** [comment]
See here it uses "$" and we should require "$$" as that'll occur less often. https://github.com/publiclab/plots2/pull/1045/files#diff-9599427925097c3c66f26ac1e0de5cadR23

This make a good first-timers-only issue!

**Comment by vincegando** [comment]
Can I take this one?

**Comment by jywarren** [comment]
We'd love your help on this and are happy to offer any assistance you might need. Thanks a lot!

Just curious -- we're trying to improve our outreach to new contributors -- how did you happen to hear about us?

**Comment by vincegando** [comment]
I heard about you through this Twitter account: https://twitter.com/first_tmrs_only

I'm having an issue when I run "passenger start". After going through the steps in the README and running that command, it says "Connecting to database specified by database.yml", then gives this error "Could not spawn process for application ../plots2: An error occurred while starting the web application: it did not write a startup response in time. ".

Any idea what's going wrong?

**Comment by jywarren** [comment]
Hmm, can you paste in your full output into http://gist.github.com and share here? 

Also, it's almost the end of the day for me, so i apologize if I'm not able to respond in a timely way -- but possibly someone from @publiclab/reviewers can assist!

Thanks!

**Comment by vincegando** [comment]
Here's the full output:
https://gist.github.com/vincegando/8f60ad079ac3be485809628bfe897eeb

I followed the instructions in the readme and made a copy of config/database.yml.sqlite.example and placed it at config/database.yml .

**Comment by jywarren** [comment]
Hmm, that's an odd error. Where are you installing this? Can you tell me a little about your system? 

If you send me your email to jeff@publiclab.org, I can get you a free https://c9.io account and we have instructions for using a Cloud9 account to get running: https://github.com/publiclab/plots2/wiki/Simple-Installation-for-Cloud9

Or, because this is a JavaScript change, I'd be comfortable accepting a change without your getting it running locally. But I def. want to help you get it running so you can solve other types of problems!

Thanks for your patience!

**Comment by vincegando** [comment]
Great! I'll send you an email and I'll also get started on this issue.

**Comment by ananyo2012** [comment]
@jywarren Welle as you see written in the documentation here http://docs.mathjax.org/en/latest/start.html#tex-and-latex-input The default delimiters are `$$..$$` but we have manually used `$..$` to enable the inline mathematics by including the line https://github.com/publiclab/plots2/pull/1045/files#diff-9599427925097c3c66f26ac1e0de5cadR23 as you have mentioned. This can be solved by simply removing that line

But keep in mind if we do so we also won't be able to use inline mathematics. Is it ok? Perhaps a better way would be to use escape sequences like `\$` for using dollar signs. This can be done by `processEscapes: true` as mentioned [here](http://docs.mathjax.org/en/latest/tex.html#tex-support). But we also have to mention this in our documentation somewhere so that people writing research notes know that. If we are comfortable using only block equations rather than inline equations then only removing that line works.

@jywarren After you decide on this I can reformat it to a `first-timers-only` issue.

**Comment by ananyo2012** [comment]
![equation2](https://cloud.githubusercontent.com/assets/10486343/21303219/c9e9a8e0-c5e2-11e6-92f8-f016d9469630.png)
Here is a snap of what is a inline and block equation. Inline equation is the one that appears above inline with text and block equation is the one that appears below and is centered.

**Comment by ananyo2012** [comment]
Ok I just missed this that; we can include inline maths with `\(..\)` So we can safely remove the `$..$` case which is a very common case.

**Comment by ananyo2012** [comment]
Closed by #1125
